.github/files/list-changed-projects.php

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Tool to list whether projects have changed.
+ *
+ * @package automattic/jetpack
+ */
+
+// phpcs:disable WordPress.WP.AlternativeFunctions, WordPress.PHP.DiscouragedPHPFunctions, WordPress.Security.EscapeOutput.OutputNotEscaped
+
+// Files that mean all tests should be run.
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+$infrastructure_files = array(
+	'.github/files/list-changed-projects.php',
+	'.github/files/generate-ci-matrix.php',
+	'.github/files/process-coverage.sh',
+	'.github/files/setup-wordpress-env.sh',
+	'.github/php-version',
+	'.github/workflows/tests.yml',
+	'.nvmrc',
+);
+
+// Globs that should be ignored. Supports `*`, `**`, and `?`.
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+$ignore_globs = array(
+	'**.md',
```

Comment:
```suggestion
	'**.md',
	'**.txt',
```

for plugin readme.txt files, at least.